Project/Assets/ML-Agents/Examples/SharedAssets/Prefabs/Targets.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
I don't think having a Targets folder is justified, it contains two almost identical prefabs : DynamicTarget and StaticTarget which are identical (except for one ☑️ )

Reply:
no changes. having 2 different prefabs lets us update all dynamic targets at one time (or static). This should make updating all dynamic or target scenes much quicker.

Reply:
I won't fight you on this, but it is possible to select multiple game objects in a scene and modify them in batches. I think there is a cost in having so many barely distinguishable prefabs.